data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+			`	He indicates a door to another room and continues, "The Stationkeeper will be out shortly. We've almost completed our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect that Kelamursh will want to travel with you. You will need to escort them to the station and dock, so they can confirm the access codes function and that there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`
+			`	As soon as he finishes, the door opens and a number of people file out, including a well-dressed Hai who immediately stops next to you.`
+			choice
+				`	(Greet the Stationkeeper.)`
+					goto greet
+				`	(Bow to the Stationkeeper.)`
+			`	She appears flattered in a Hai sort-of way. "There is no need, Captain, my station is not operational, and this is a business operation, not an appointed position. For now, anyway."`
+			label greet
+			`	You introduce yourself. "I am Kelamursh, but you should refer to me as Stationkeeper," she says in response. "Now, if you don't mind, I believe your arrival has solved our last remaining problems, putting us in good stead to depart momentarily. If I may see your ship's accommodation we'll make final preparations and depart in an hour or so."`
+			choice
+				`	"Let's go."`
+					goto depart
+				`	"We were getting paid for this, right?"`
+			`	She blinks in surprise. "Of course, didn't they tell you? Nevermind, you can confirm the details with Heyru after I've checked your vessel."`
+			label depart
+			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. With the concerns on Allhome, she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`
+
+
+mission "HRCP: Sandsnap Mercenaries"
+	landing
+	invisible
+	source "Makerplace"
+	to offer
+		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
+	to complete
+		never
+	npc
+		government "Hai Merchant (Human)"
+		personality escort coward opportunistic vindictive
+		fleet
+			names "hai (human)"
+			variant
+				"Grasshopper (Ion)"
+				"Grasshopper (Tracker)"
+				"Headhunter (Hai)" 2
+				"Firebird (Hai Shields)"
+				"Firebird (Hai Weapons)"
+				"Lightning Bug (Pulse)"
+				"Corvette (Hai)"
+				"Leviathan (Hai Weapons)"
+
+
+mission "Hai Reveal Concern Period: Securing Sandsnap"
```

Comment:
All I had to do was take off, wait for the fleet to arrive, then land.

Reply:
That is a possible outcome, yes.

Reply:
I don’t think it should be, if your goal is to protect the fleet. I know that escort missions do this, but they usually require at least traveling. This could be solved by requiring the fleet to be killed, now that you actually get your NPC escorts, but I’m not sure how often that would just lead to frustration when the ships that must be saved decide to sacrifice themselves fighting pirates, failing the mission.

It felt anti-climatic landing, but then again if I had NPC escorts, which I should now have, I probably would have tried to fight unless doing so became frustrating, and there are probably others who don’t want to deal with this at all, however, those players would have just declined the mission.

So, I’m still leaning towards imposing some additional requirement, but I can understand if you decide to keep it the way it is.

Reply:
Last time I checked there's a fairly high likelihood of having to fight something. In this instance I would say you got lucky.

Reply:
OK, that makes sense. Thank you.